ext/nmatrix/ruby_nmatrix.c

```diff
@@ -3006,12 +3006,12 @@ static VALUE nm_det_exact(VALUE self) {
   nm::dtype_t dtype = NM_DTYPE(self);
   nm_math_det_exact(NM_SHAPE0(self), NM_STORAGE_DENSE(self)->elements, NM_SHAPE0(self), NM_DTYPE(self), result);
 
+  VALUE to_return;
   if (dtype == nm::RUBYOBJ) {
```

Comment:
Good catch on this. @cjfuller Just want to check this with you since you wrote the register/unregister functions.